src/index.js

```diff
@@ -1,29 +1,14 @@
+/* eslint-disable import/no-cycle, object-curly-newline, import/no-mutable-exports */
```

Comment:
I think that disabling multiple linter rules is a bad practice. The purpose of them is to keep our code clean, avoid issues and standardize code between teams. For that reason, it's important that we research anytime we're facing these to implement a better solution than just disabling them.

- [ ] Let's try to remove as much of them as possible, especially the second one and third one I think are easy to fix. The first one is a bit more tricky and requires a lot of refactor, if you can't fix it it's okay.